src/components/Carousel.jsx

```diff
@@ -13,7 +13,9 @@ export default function Carousel() {
     arrows: true,
   };
 
-  const { data, error, isLoading } = useGetAllVespasQuery();
+  const { data:vespas, error, isLoading } = useGetAllVespasQuery();
+
+    console.log(vespas)
```

Comment:
- [optional] please remove the extra console log to make your console cleaner 🧹

Reply:
Done